jobs/build/refresh-images/Jenkinsfile

```diff
@@ -24,33 +24,141 @@ ${OSE_MAJOR}.${OSE_MINOR}
 """);
 }
 
+//
+// Search the build log for 
+//
+def get_failed_builds(log_dir) {
+    record_log = buildlib.parse_record_log(log_dir)
+    builds = record_log['build']
+    failed_map = [:]
+    for (i = 0; i < builds.size(); i++) {
+        bld = builds[i]
+        distgit = bld['distgit']
+        if (bld['status'] != '0') {
+            failed_map[distgit] = bld['task_url']
+        } else if (bld['push_status'] != '0') {
+            failed_map[distgit] = 'Failed to push built image. See debug.log'
+        } else {
+            // build may have succeeded later. If so, remove.
+            failed_map.remove(distgit)
+        }
+    }
+
+    return failed_map
+}
+
+KRB5_KEYTAB_FILE =
+    "/home/jenkins/ocp-build-buildvm.openshift.eng.bos.redhat.com.keytab"
+KRB5_PRINCIPAL = "ocp-build/buildvm.openshift.eng.bos.redhat.com@REDHAT.COM"
+
+def get_kerberos_ticket() {
```

Comment:
buildlib already has this:
https://github.com/openshift/aos-cd-jobs/blob/ae87bb9f2d9a8a38fe3147ba2e2a2508d32dddd2/pipeline-scripts/buildlib.groovy#L29